test/runtests.jl

```diff
@@ -40,6 +40,14 @@ Distributed.addprocs(5)
         end
     end
 
+    @testset "deprecated tags syntax" begin
+        mktempdir() do path
+            Checkpoints.config("TestPkg.deprecated", path)
+            TestPkg.deprecated_checkpoint_syntax()
```

Comment:
I think you can use `@test_deprecated` to check this, and also maybe surpress the warning.
I haven't tried yet
```suggestion
            @test_deprecated TestPkg.deprecated_checkpoint_syntax()
```